WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1506,20 +1655,58 @@ private void showDialogError(@StringRes int resIdMessage, @StringRes int resIdPo
         dialog.show(getSupportFragmentManager(), tag);
     }
 
-    private void showDialogProgress(boolean show) {
-        if (show) {
-            mProgressDialog = new ProgressDialog(this);
-            mProgressDialog.setCancelable(false);
-            mProgressDialog.setIndeterminate(true);
-            mProgressDialog.setMessage(mIsDiscardingChanges
-                    ? getString(R.string.local_changes_discarding)
-                    : getString(R.string.history_loading_revision));
-            mProgressDialog.show();
-        } else if (mProgressDialog != null) {
-            mProgressDialog.dismiss();
+    private void setPreviewingInEditorSticky(boolean enable, @Nullable PostModel post) {
+        if (enable) {
+            if (post != null) {
+                EventBus.getDefault().postSticky(
+                        new PostEvents.PostPreviewingInEditor(post.getLocalSiteId(), post.getId()));
+            }
+        } else {
+            PostEvents.PostPreviewingInEditor stickyEvent =
+                    EventBus.getDefault().getStickyEvent(PostEvents.PostPreviewingInEditor.class);
+            if (stickyEvent != null) {
+                EventBus.getDefault().removeStickyEvent(stickyEvent);
+            }
         }
+    }
 
-        mIsDialogProgressShown = show;
+    private void updatePostLoadingAndDialogState(PostLoadingState postLoadingState) {
+        updatePostLoadingAndDialogState(postLoadingState, null);
+    }
+
+    private void updatePostLoadingAndDialogState(PostLoadingState postLoadingState, @Nullable PostModel post) {
+        /* We need only transitions, so... */
+        if (mPostLoadingState == postLoadingState) return;
+
+        AppLog.d(
+                AppLog.T.POSTS,
+                "****** EDITOR PREVIEW SM - from [" + mPostLoadingState + "] to [" + postLoadingState + "]"
```

Comment:
Same comment as above about the log.

Reply:
For same reason as above I would keep it reformatting. Done in same commit as above d3958f7e6bd55072e65c999c3f0ebbf3e303a486